src/MSBuild/XMake.cs

```diff
@@ -4269,7 +4269,7 @@ private static LoggerDescription ParseLoggingParameter(string parameter, string
             }
             catch (Exception e) when (loggerDescription.IsOptional)
             {
-                Console.WriteLine(ResourceUtilities.FormatResourceStringStripCodeAndKeyword("OptionalLoggerCreationMessage", e.Message));
```

Comment:
What about all the messages from the catch blocks above - also for logger creation failed?

Reply:
Yes, we should feed it in there too.

Reply:
@yanrez - do you plan to address this? 
Once it's adressed then this change is ready to go.

Reply:
I filed a bug to track this and don't think it's blocking this PR.